tests/test_config_edge_cases.py

```
@@ -314,6 +323,239 @@ def test_load_config_no_files_found(self):
+                        # Should use base_dir/store with override
+                        self.assertEqual(result, "C:\\mmrelay\\store")
+
+    def test_load_credentials_windows_debug(self):
+        """Test load_credentials on Windows logs directory contents."""
+        with patch("mmrelay.config.sys.platform", "win32"):
+            with patch("mmrelay.config.os.path.exists", return_value=False):
+                with patch("mmrelay.config.get_base_dir", return_value="C:\\mmrelay"):
+                    with patch(
+                        "mmrelay.config.os.listdir",
+                        return_value=["file1.txt", "file2.json"],
+                    ) as mock_listdir:
```

Ruff (0.14.14)

[error] Local variable `mock_listdir` is assigned to but never used

Remove assignment to unused variable `mock_listdir`

(F841)